fakturogen-invoice/src/test/java/pl/fakturogen/invoice/service/mapper/InvoiceMapperTest.java

```diff
@@ -0,0 +1,131 @@
+package pl.fakturogen.invoice.service.mapper;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import pl.fakturogen.invoice.dao.entity.Customer;
+import pl.fakturogen.invoice.dao.entity.Invoice;
+import pl.fakturogen.invoice.dao.entity.Product;
+import pl.fakturogen.invoice.dao.entity.Rate;
+import pl.fakturogen.invoice.web.dto.InvoiceSaveDTO;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("Invoice ModelMapper ")
+class InvoiceMapperTest {
+    InvoiceMapper invoiceMapper;
+    List<Product> items;
+    Customer customer;
+
+    @BeforeEach
+    void init() {
+        invoiceMapper = new InvoiceMapper();
+        customer = new Customer();
+        items = new ArrayList<>();
+    }
+
+    @DisplayName(" - should map InvoiceSaveDTO to entity")
+    @Test
+    public void test1() {
+        Product product1 = new Product();
+        product1.setId(1);
+        product1.setRate(Rate.R9);
+        product1.setSaleNetPrice(100.00);
+        product1.setName("Apartment no. 1");
+
+        Product product2 = new Product();
+        product2.setId(12);
+        product2.setRate(Rate.R9);
+        product2.setSaleNetPrice(100.00);
+        product2.setName("Apartment no. 2");
+
+        items.add(product1);
+        items.add(product2);
+
+        customer.setName("Jan");
+        customer.setId(4L);
+        customer.setCustomerTaxNumber("123");
+
+        InvoiceSaveDTO receivedDTO = InvoiceSaveDTO.builder()
+                .number("1/1/2021")
+                .status(0)
+                .issueDate(LocalDate.of(2021, 1, 19))
+                .dueDate(LocalDate.of(2021, 1, 26))
+                .paymentMethod(1)
+                .total(246.00)
+                .tax(0.23)
+                .net(200.00)
+                .discount(00.0)
+                .status(1)
+                .customer(customer)
+                .items(items)
+                .bankAccountId(1234)
+                .invoiceType(0)
+                .additionalInformation("brak")
+                .originalId(23L)
+                .build();
+
+        Invoice mappedInvoice = new Invoice();
```

Comment:
This mappedInvoice field is probably not needed as I can't see where you use it for test case in code.